package.json

```diff
@@ -53,8 +53,8 @@
     "prettier-standard": "^8.0.1",
     "standard": "^11.0.1",
     "tape": "^4.9.0",
-    "webpack": "^4.8.3",
-    "webpack-cli": "^2.1.3",
+    "webpack": "^4.20.2",
```

Comment:
Why is it part of this PR?

Reply:
Commands calling webpack will not run anymore with the previous version because of this issue: https://github.com/webpack/webpack/issues/8082

Reply:
For the sake of https://hacktoberfest.digitalocean.com/ I should split this into more pull requests ;-)